### ISSUE TITLE
styled-components missing from library when components are used in an application

### ISSUE THREAD

**Issue by ttwebagency** [ttwebagency/react-component-library#9]
When the library is being installed and used in a React application, an error is flagged showing that `styled-components` is missing. This is listed as a `devDependency` when it should likely be listed as a `dependency`.

For the moment, the package has to be installed within the React application where the library is being used:

```
npm install styled-components
```

**Comment by ttwebagency** [comment]
Installed the `styled-components` package as a `dependency` in version `0.2.1`. This is to be tested before this issue/bug can be marked as fixed.

**Comment by ttwebagency** [comment]
This was tested with version `0.5.3` in the test application and the issue no longer appears. Consider this bug to be fixed in version `0.2.1`.

Styling from the `styled-components` is injected into the `<head>` section of the React application HTML code within a `<style>` tag:

```html
<style data-styled="active" data-styled-version="5.3.5"> ... </styled>
```

**Comment by ttwebagency** [comment]
Issue has been closed. Fix applied in version `0.2.1`.
